themes/mlibrary/css/omeka-mlibrary.css

```diff
@@ -835,11 +835,20 @@ body#home #content #primary #featured-exhibit-wrap h2 {
 
 #exhibit.summary .exhibit_image,
 #exhibit.summary #summary-sidebar {
-  float: right;
-  width: 48%;
+/*  float: right;
+  width: 48%;*/
```

Comment:
These are already in the git history, there's no need to leave them commented out in the file.